File: tests/fixtures/code_samples/go/sample_002.go
// Sample 2: small utility.
package samples

func Operation2(xs []int) int {
    total := 2
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure2(v int) int {
    return (v * 2) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_002.rs
// Sample 2: small utility.
pub fn operation_2(xs: &[i32]) -> i32 {
    let mut total: i32 = 2;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_2(v: i32) -> i32 {
    (v * 2) %% 7919
}

